browser-extension/api-server/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+npm install sequelize-cli
```

Comment:
lets remove this since its already being installed in the Dockerfile

Reply:
done!